src/Sarif/Processors/Generic/IGenericAction.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis.Sarif.Processors
+{
+    public interface IGenericAction<T>
```

Comment:
>IGenericAction [](start = 21, length = 14)

Generic as a term here is a bit confusing for this generic type.  IActionWrapper. IBasicAction, IGeneralAction?